test/lib/build/ProjectBuilder.js

```diff
@@ -75,7 +75,7 @@ test.afterEach.always(() => {
 	mock.stopAll();
 });
 
-test("Missing graph parameters", async (t) => {
+test.serial("Missing graph parameters", (t) => {
```

Comment:
Is there a reason why `serial` is added for some tests?

Reply:
Currently, other tests are failing when I remove the `async` identifier from the test function. The `serial` functions fixes this order issue. But I was not able to identify why the other test has any connection to these tests

Reply:
The problem seems to be in the `afterEach` handler. It uses the global sinon sandbox to restore stubs, which affects also other tests. So either all tests should use serial or (better) they should use a local sinon sandbox that is the reset in the afterEach. I think we used this already once somewhere (instead of making all tests serial which is more common in our tests as it needs to be done anyways when shared modules are mocked/stubbed).

Reply:
Refactored by using the sandbox concept.